kolaintro.md

```diff
@@ -0,0 +1,81 @@
+ # Kola Introduction Documentation
+ 
+ This document just goes through the basics of how kola test's work and how someone just beginning can get started
+ 
+
+
+ 
+ #### Steps to run kola
+
+
+1. mkdir src
+2. git clone https://github.com/openshift/os src/config
+3. pushd src/config
+4. git fetch --all
+5. git checkout origin/master
+6. git submodule update --init --recursive
+7. cd /src
+8. git clone --branch master https://gitlab.cee.redhat.com/coreos/redhat-coreos.git src/tmp
+9. cp -Rf src/tmp/*.repo src/config/
+10. coreos-assembler init --force /srv/config
+11. cosa fetch
+12. cosa build
+13. cosa buildextend-metal
+    
+
+
+
+* `cosa kola run`
+* `cosa kola --parallel run`
+* `cosa kola basic`
+
+### Versions older than 4.9
+
+1. cd /home/builder 
+2. mkdir src
+3. git clone https://gitlab.cee.redhat.com/coreos/redhat-coreos.git src/config
+4. pushd src/config
+5. git fetch --all
+6. git checkout origin/master
+7. git submodule update --init --recursive
+8. cd /home/builder 
+9. coreos-assembler init --force /home/jenkins/agent/workspace/rhcos/rhcos-rhcos-4.8/src/config
+10. cosa fetch
+11. cosa build
+12. cosa buildextend-metal
+13. cosa buildextend-metal4k
+14. cosa buildextend-live
+15. cosa buildextend-installer
+16. cosa kola run --basic-qemu-scenarios
+17. cosa kola run -d --parallel 3
```

Comment:
This is good information, thought it is still targeting RHCOS and has links to internal gitlab repos.

Can you retool these steps to build + test FCOS, that way we can present a more generic approach to running tests and triaging failures.

Reply:
Additionally, some of the information about building CoreOS is replicated with existing docs - https://coreos.github.io/coreos-assembler/building-fcos/

I would perhaps refocus the content of this doc around building extended artifacts (metal, metal4k, live ISO, etc) and running tests against them.  You can point folks to the existing docs to get started and then start with the assumption that users already have `ostree` and `qemu` built.